webpack.config.ts

```diff
@@ -69,9 +69,11 @@ const serverConfig = (
 			? []
 			: [
 					nodeExternals({
-						allowlist: [/@guardian/],
+						allowlist: [/@guardian/, 'React'],
```

Comment:
What does this do?

Reply:
Not sure if this was automatically done as part of the upgrade

Reply:
What happens if you remove it?

Reply:
It may have been something i tried last week? cant remember, omitting it still passes the tests locally. Should we take it out?

Reply:
> What happens if you remove it?

passes local tests and run watch no problems

Reply:
If it doesn't cause any errors I don't think you need it. Adding here is saying "when in development, please include React in my Webpack bundle".

Reply:
👍